js/CutScene.js

```diff
@@ -45,14 +49,13 @@ Critterer.CutScene.prototype = {
               href          : this.valueToUse,
               helpers       : {
                   media : {}
-              }
-     });    
-   },
-    
+              },
```

Comment:
commas are not to be used after the last element in a javascript set, in this case, `helpers : { ... },`, remove the comma.